lib/ai-stuff/analyze-discussion.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from "zod"
+import type { DiscussionComment, DiscussionContribution } from "lib/types"
+import { generateAiObjectCached } from "./sdk"
+import { generateAnalyzeDiscussionPrompt } from "./prompts"
+
+const discussionSchema = z.object({
+  level: z.enum(["Participating", "VeryActive", "ExtremelyActive"]),
```

Comment:
just saying, this doesn't make sense
```suggestion
  level: z.enum(["NormalComment", "GreatInformativeComment", "IncredibleCommentTopTier"]),
```

Reply:
Discussions comments

Reply:
yea so i think we should change the enum to my suggestion (in future PR etc. not the point of this PR)